tests/test_adjoint.py

```diff
@@ -58,7 +57,7 @@ def test_adjoint_F(self, mkey, shape, kernel, space_order):
 
         # Adjoint test: Verify <Ax,y> matches  <x, A^Ty> closely
         term1 = np.dot(srca.data.reshape(-1), solver.geometry.src.data)
-        term2 = linalg.norm(rec.data.reshape(-1)) ** 2
+        term2 = norm(rec) ** 2
```

Comment:
note that this is causing more and more codegens and compiles, thus it also affects the CI turnaround time

Reply:
Not by much